federation/pkg/federation-controller/secret/secret_controller.go

```diff
@@ -368,6 +368,11 @@ func (s *FederationSyncController) reconcile(namespacedName types.NamespacedName
 	}
 	obj := copiedObj.(pkgruntime.Object)
 	meta := s.adapter.ObjectMeta(obj)
+	if util.IsFederationOnlyObject(obj) {
+		// Should not reconcile federation-only objects
```

Comment:
This comment duplicates the content of the log message - please remove (everywhere)